src/radiant.c

```diff
@@ -2920,25 +2956,27 @@ int radiant_get_delays(radiant_dev_t * bd,radiant_readout_delay_t * rd)
   return 0;
 }
 
-//closer to hardware level of how delays and masks are appiled
+//firmware level read from FPGA memory in case this ever needs to be called
 int radiant_get_delay_settings(radiant_dev_t * bd, uint8_t * rf0_delay, uint8_t * rf1_delay, uint8_t * rf0_delay_mask,uint8_t * rf1_delay_mask)
 {
+
   if(bd->rad_dateversion_int<=230)
   {
     printf("wrong fpga version - delays(and fpga memory) not implemented!!!\n");
```

Comment:
Do we want this printout, or just have it be a no op? I agree on setting delays, it should warn that it's not possible  with old firmware, but otherwise the delays are all 0, which is perfectly valid?

Reply:
Also, this may not need to be exposed to the public API, since I don't think there is a use case?

Reply:
this function should get removed and its contents get placed in radiant_open... it was useful in testing the c implementation of the delays to make sure I was reading and writing correctly. but now not so useful